server/text_generation_server/models/causal_lm.py

```diff
@@ -643,6 +643,7 @@ def __init__(
         else:
             self.hb_profiler = None
         self.step = 0
+        self._shifting_warmup_done = False
```

Comment:
I guess this flag is not needed

Reply:
I added it only for future purposes, but can remove